k2/python/tests/ragged_shape_test.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
```

Comment:
Please add this file to `k2/python/tests/CMakeLists.txt`.
https://github.com/k2-fsa/k2/blob/048da2a6f7fce1704f39a76c8dd5eef0b9b613fd/k2/python/tests/CMakeLists.txt#L18-L20

Reply:
fixed.